# Left bat position (y)
float64 left_pos_y
# Right bat position (y)
float64 right_pos_y

